<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>524</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>675</width>
    <height>524</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>675</width>
    <height>524</height>
   </size>
  </property>
  <property name="font">
   <font>
    <underline>true</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Find text on WEB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>247</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="bStart">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myresources.qrc">
         <normaloff>:/data/res/start.png</normaloff>:/data/res/start.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bPause">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myresources.qrc">
         <normaloff>:/data/res/pause.png</normaloff>:/data/res/pause.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bStop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myresources.qrc">
         <normaloff>:/data/res/stop.png</normaloff>:/data/res/stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>N(Proc)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="eProcCount">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>N(Pages)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ePageCount">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="myresources.qrc">:/data/res/Developex.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>TextFinder</string>
    </property>
   </widget>
   <widget class="QTableView" name="tFoundURLs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>321</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="eURL">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="eText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Text </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>URL </string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
